spring-xd-dirt/src/main/java/org/springframework/xd/dirt/stream/ZKStreamDeployer.java

```diff
@@ -18,19 +18,12 @@
 
 import static org.springframework.xd.dirt.stream.ParsingContext.stream;
 
-import java.util.ArrayList;
-import java.util.List;
-
-import org.springframework.xd.dirt.stream.dsl.StreamDefinitionException;
-import org.springframework.xd.dirt.stream.dsl.XDDSLMessages;
+import org.springframework.xd.dirt.server.admin.deployment.DeploymentHandler;
 import org.springframework.xd.dirt.zookeeper.Paths;
 import org.springframework.xd.dirt.zookeeper.ZooKeeperConnection;
-import org.springframework.xd.module.ModuleDefinition;
-import org.springframework.xd.module.ModuleDefinitions;
-import org.springframework.xd.module.ModuleDescriptor;
 
 /**
- * Default implementation of {@link StreamDeployer} that uses provided
+ * Default implementation of {@link ZKStreamDeployer} that uses provided
```

Comment:
This javadoc comment in not in synch with the class. It is self-referencing.

Also, why the rename to ZK? Nothing in the contract leaks ZK, does it?